tests/unit/test_configuration.py

```diff
@@ -233,28 +248,59 @@ def test_overriding_nonexistent_keys_fails(config, match):
         get_configuration(config)
 
 
+def get_noise_type_configs(noise_names: list):
+    configs = list(itertools.product(noise_names, PROBABILITY_VALUE_LOGS))
+    return [(x[0], x[1][0], x[1][1]) for x in configs]
+
+
 @pytest.mark.parametrize(
-    "value, match",
-    [
-        ("a", "must be floats or ints"),
-        (-0.01, "must be between 0 and 1"),
-        (1.01, "must be between 0 and 1"),
-    ],
+    "row_noise_type, value, match",
+    get_noise_type_configs(ROW_NOISE_TYPES),
+)
+def test_validate_standard_parameters_failures_row_noise(row_noise_type, value, match):
+    """
+    Tests valid configuration values for probability for row noise types.
+    """
+    dataset_name = [
+        dataset for dataset in DATASETS if row_noise_type in dataset.row_noise_types
+    ][0].name
```

Comment:
This and the similar call below that gets a column are just grabbing the first valid dataset (or column below) from the list so we can test that specific noise type.